rest-api/alembic/versions/b315fec9aa4e_adding_pm_view.py

```diff
@@ -26,13 +26,13 @@
    p.withdrawal_time participant_withdrawal_time,
    p.suspension_status participant_suspension_status,
    p.suspension_time participant_suspension_time,
-   pm.physical_measurements_id physical_measurements_id,   
+   pm.physical_measurements_id physical_measurements_id,
```

Comment:
These look like they are just whitespace changes, but should mutating these alembic migration files be generally avoided as a rule after initial submission?

Reply:
Modifying their contents in a significant way should be avoided, yes. Modifying whitespace is no big deal.

(The remove_trailing_whitespace script is what did this, BTW.)